queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py

```diff
@@ -153,6 +153,9 @@ def update_parties(business: Business, parties: dict, correction_filing_rec: Fil
         # Create if id not present
         # If id is present and is a GUID then this is an id specific to the UI which is not relevant to the backend.
         # The backend will have an id of type int
+        role_type = party_info.get('roleType', None)
+        if is_coop and is_sr_correction and role_type != 'Completing Party':
+            continue
```

Comment:
this makes sense but the code below for creating or updating the party will be incorrect.  It will be creating a party entry that is tied to the business.  It should be tied to the filing.  

I think you will want to reference the IA filing processor to see how completing parties are attached to a filing.
https://github.com/bcgov/lear/blob/911e3976ace27461a05fc723fa5079e81438a612/queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py#L144-L145
https://github.com/bcgov/lear/blob/475cde8fcb7f507adf1b97e47d8cb3f2e2cbf470/queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/parties.py#L59-L61




For example, this is what completing party looks like in the db for the IA filing.

![image](https://github.com/bcgov/lear/assets/6685223/b1a4259a-079c-4a55-80da-46db58cc47ab)

Reply:
My fault, linked her the wrong code

Reply:
updated `def _create_partty_info`:
```
        if _is_cp_sr_completing_party(business, role_str, is_sr_correction):
            continue
        if party_role.role in [PartyRole.RoleTypes.COMPLETING_PARTY.value]:
            correction_filing_rec.filing_party_roles.append(party_role)
```